google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/GrpcResultSetTest.java

```diff
@@ -665,4 +665,160 @@ private void verifySerialization(
     Struct copy = reserialize(row);
     assertThat(row).isEqualTo(copy);
   }
+
+  @Test
```

Comment:
general comment across all:
1. easier to read test with names being explicit
e.g. timestampArray, longArray etc.. 

2. assert resultSet.next() is true while reading each result & false after the expected values are tested

Reply:
Will change variable names. For test names do you mean getBoolean() etc. is not clear? How about getBooleanSucceeds()?

Reply:
I've added assert on resultSet.next(). I do feel however that it's distracting since this is not what this testcase is testing. i.e it's not testing that resultset.next() is correct. It's testing the types are being returned with the correct values.

However, I don't feel strongly either way, so I changed it to what you want.

Reply:
1. test names look fine
2. this is just for sanity check.